src/helm/benchmark/static/schema.yaml

```diff
@@ -1928,6 +2039,20 @@ run_groups:
       main_name: none
       main_split: none
 
+  - name: decodingtrust
+    display_name: DecodingTrust
+    description: A comprehensive benchmark of the trustworthiness of large language models [(Wang et. al. 2023)](https://decodingtrust.github.io/)
+    category: Core scenarios
+    subgroups:
+      - adv_robustness
+      - adv_demonstration
+      - ood_robustness
+      - fairness
+      - privacy
+      - machine_ethics
+      - decodingtrust_toxicity_prompts
+      - stereotype_bias
```

Comment:
These need to match the group names below e.g. `adv_robustness` should be `decodingtrust_adv_robustness`, `decodingtrust_adv_demonstration` should be `decodingtrust_adv_demonstration`, etc.

Reply:
Sorry for the oversight. This has been fixed in 9006278158394933f8eca438d57ea9083866ac65.